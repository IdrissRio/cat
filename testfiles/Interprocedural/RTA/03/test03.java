public class test03 {
  public void m() {}
}

class Internal {
  void q() { new test03().m(); }
}
