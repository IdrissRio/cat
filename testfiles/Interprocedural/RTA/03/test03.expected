test03::Static:{
}
test03::Instance:{
}
test03::m:{
}
Internal::Static:{
}
Internal::Instance:{
}
Internal::q:{
	test03::m
}
